workflows/reference_files/2.4/reference_files.smk

```diff
@@ -385,6 +385,20 @@ rule get_af_only_gnomad_vcf:
         tabix {output.vcf}
         """)
 
+rule normalize_af_only_gnomad_vcf:
+    input:
+        fasta = rules.get_genome_fasta_download.output.fasta,
+        vcf = str(rules.get_af_only_gnomad_vcf.output.vcf)
+    output:
+        vcf = "genomes/{genome_build}/variation/af-only-gnomad.normalized.{genome_build}.vcf.gz"
```

Comment:
Can you make the tabix index an output also?

Reply:
Sure, I have specified the tabix index as rule output here